lighthouse-cli/test/smokehouse/test-definitions/core-tests.js

```diff
@@ -5,77 +5,77 @@
  */
 'use strict';
 
-/** @type {Array<Smokehouse.TestDfn>} */
+/** @type {ReadonlyArray<Smokehouse.TestDfn>} */
 const smokeTests = [{
   id: 'a11y',
   expectations: require('./a11y/expectations.js'),
   config: require('./a11y/a11y-config.js'),
-  batch: 'parallel-first',
+  runParallel: true,
```

Comment:
> maybe rename to `runInBand`, and have the default value be `false` so it can be excluded?

yeah, good idea, several of these seemed to have been set to running serially when it doesn't seem necessary, so better to have to put extra work in to make a new test not run in parallel :)